test/images/iperf/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=gcr.io/google_containers/ubuntu-slim:0.12
```

Comment:
`debian-base` is probably preferred, yes, though it matters less for test images where security isn't the primary concern.

fine to follow-up in a later PR.